detection_rules/misc.py

```diff
@@ -32,6 +55,291 @@
 """.strip().format("\n".join(' * ' + line for line in LICENSE_LINES))
 
 
+def get_gh_release(repo: Repository, release_name=None, tag_name=None) -> GitRelease:
+    """Get a list of GitHub releases by repo."""
+    assert release_name or tag_name, 'Must specify a release_name or tag_name'
+
+    releases = repo.get_releases()
+    release = next((r for r in releases
+                    if release_name and release_name == r.title or tag_name and tag_name == r.tag_name), None)
```

Comment:
i know it's redundant, but it doesn't hurt
```suggestion
                    if (release_name and release_name == r.title) or (tag_name and tag_name == r.tag_name)), None)
```